Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -2952,9 +3041,12 @@ def main():
 
     IS_VERSION_2_1 = api_version == '2.1'
 
+    fetch_type = params.get('fetch_type')
     first_fetch_time = params.get('fetch_time', '3 days')
+    fetch_severity = params.get('fetch_severity', 'High')
```

Comment:
this param is multiple select
```suggestion
    fetch_severity = params.get('fetch_severity', [])
```